### PR TITLE
[DC-3132] Update `MissingConceptRecordSuppression` to include `aou_death`

### PR DESCRIPTION
**Pull request by hiro-mishima**
Dear reviewer,
Please reivew DC-3224 first -> https://github.com/all-of-us/curation/pull/1578

### CODE REVIEW

File: tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/missing_concept_record_suppression_test.py [hunk 1]
```diff
@@ -96,35 +121,37 @@ def test_missing_concept_record_suppression_cleaning(self):
...
-                (1, 1, 4225432, self.date, 441840, 441840, 441840, 441840,
-                 441840, 441840),
-                (2, 2, 4297377, self.date, 441840, 441840, 441840, 441840,
-                 441840, 441840),
-            ]
+            'fields': ['observation_id'],
+            'cleaned_values': [(1,), (2,), (5,), (6,)]
         }]
 
         self.default_test(tables_and_counts)
```
Comment (MichaelSchmidt82) [inline tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/missing_concept_record_suppression_test.py:157]:
```suggestion
        self.default_test(tables_and_counts)
        
```

Comment (MichaelSchmidt82) [reply tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/missing_concept_record_suppression_test.py:157]: The irony is that you stripped out all that trailing whitespace, except the very [last line](https://gist.github.com/OleksiyRudenko/d51388345ea55767b7672307fe35adf3#why-should-text-files-end-with-a-newline).

Comment (hiro-mishima) [reply tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/missing_concept_record_suppression_test.py]: I knew it is better to end with a new line, but I did not know why till right now! :sunglasses:

**Review by hiro-mishima — COMMENTED** [review]
@MichaelSchmidt82 Thanks for reviewing this PR! 
Would you mind also taking a look at DC-3224 -> https://github.com/all-of-us/curation/pull/1578? DC-3132 depends on DC-3224, so this PR has to be merged after DC-3224 is merged.

**Comment by hiro-mishima** [comment]
@MichaelSchmidt82 @brendagutman DC-3224 (https://github.com/all-of-us/curation/pull/1578) is merged to `develop`, so this PR now has only the changes relevant to DC-3132. It's ready for review. Thanks!

**Review by MichaelSchmidt82 — APPROVED** [review]

**Pull request merged**